pkg/runit/servicebuilder.go

```diff
@@ -113,6 +117,19 @@ __END__
 	return []byte(ret), nil
 }
 
+func (s ServiceTemplate) finishScript() ([]byte, error) {
+	finish_exec := s.Finish
+	if len(finish_exec) == 0 {
+		finish_exec = []string{"/bin/true", "# finish not implemented"}
```

Comment:
I'm guessing not, but nevermind because it's just interpolated into a real bash script below